ocdaction/tasks/views.py

```diff
@@ -6,14 +6,21 @@
 
 
 @login_required
-def task_list(request):
+def task_list(request, archived=None):
     """
     Displays a list os user tasks on ACT view
     """
     template_name = "dashboard/act/task_list.html"
-    tasks = Task.objects.filter(user=request.user, is_archived=False).order_by('-created_at', '-updated_at')[:10]
 
-    return render(request, template_name, {'tasks': tasks})
+    if archived == None:
+        tasks = Task.objects.filter(user=request.user, is_archived=False).order_by('-created_at', '-updated_at')[:10]
+        context = {'tasks': tasks}
+    else:
+        tasks = Task.objects.filter(user=request.user, is_archived=True).order_by('-created_at', '-updated_at')[:10]
+        context = {'tasks': tasks, 'is_archive': True}
```

Comment:
A bit of a nitpick :) what do you think about changing `is_archive` to `archived` for the code readability? I feel it would read better if in templates if we'd use 
` {% if archived %}`
`{% if not archived %}`
instead of 
` {% if is_archive %}`
`{% if not is_archive %}`